rules/sinks/third_parties/sdk/knadh/go.yaml

```diff
@@ -0,0 +1,13 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
```

Comment:
Verify if this library deals with PII @ankit-privado

Reply:
Here is one use case I found https://github.com/knadh/koanf#reading-from-an-s3-bucket.

Reply:
@hiteshbedre

Reply:
resolved.